bus-mapping/src/circuit_input_builder/tracer_tests.rs

```diff
@@ -246,25 +246,75 @@ fn tracer_err_insufficient_balance() {
     let next_step = block.geth_traces[0].struct_logs.get(index + 1);
     assert_eq!(step.error, None);
     assert_eq!(next_step.unwrap().op, OpcodeId::PUSH2);
-    assert_eq!(next_step.unwrap().stack, Stack(vec![Word::zero()])); // success = 0
+    assert_eq!(next_step.unwrap().stack, Stack(vec![Word::zero()])); // failure = 0
```

Comment:
hmm I think the value on stack means `success`.

Reply:
@han0110 I understand the stack value is for call result in https://github.com/ethereum/go-ethereum/blob/master/core/vm/instructions.go#L698. in this case call's `ret` is zero, right ?

Reply:
Yes